var/spack/repos/builtin/packages/apple-opengl/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class AppleOpengl(Package):
+    """Placeholder package for Apple OpenGL"""
+
+    homepage = "https://developer.apple.com/library/archive/documentation/GraphicsImaging/Conceptual/OpenGL-MacProgGuide/opengl_intro/opengl_intro.html"
+
+    maintainers = ["aphecetche"]
+
+    has_code = False
+
+    version("4.0.0")
+
+    provides("gl", when=sys.platform == "darwin")
+
+    # Only supported on 'platform=darwin' and compiler=apple-clang
+    conflicts("platform=linux")
+    conflicts("platform=cray")
+    conflicts("%gcc")
+    conflicts("%clang")
+
+    phases = []
+
+    def setup_dependent_build_environment(self, env, dependent_spec):
+        # we try to setup a build environment with enough hints
+        # for the build system to pick up on the Apple framework version
+        # of OpenGL.
+        # - for a cmake build we actually needs nothing at all as
+        # find_package(OpenGL) will do the right thing
+        # - for the rest of the build systems we'll assume that
+        # setting the C_INCLUDE_PATH will be enough for the compilation phase
+        # and *** for the link phase.
+        env.prepend_path(
+            "C_INCLUDE_PATH",
+            (
+                "/Applications/Xcode.app/Contents/Developer/",
+                "Platforms/MacOSX.platform/Developer/SDKs/MacOSX",
+            ),
+        )
+        # env.set("SPACK_APPLE_OPENGL","1")
+
+    @property
+    def headers(self):
+        return HeaderList(
+            (
+                "/Applications/Xcode.app/Contents/Developer/",
+                "Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/",
+                "Library/Frameworks/OpenGL.framework/Headers",
+            )
+        )
```

Comment:
I only tested this within the `root` recipe, which uses CMake's `find_package(OpenGL)` and CMake was happy as is. Probably because the `glu.h` is in fact colocated with the `gl.h` one ?
Anyway I can certainly add it, yes.